src/components/ButtonGroup/ButtonGroup.tsx

```diff
@@ -18,7 +18,7 @@ export interface Props {
    * - **responsive** results in a ButtonGroup that stacks on small screens, but displays side by side on larger screens
    * - **stacked** results in a ButtonGroup that stacks on all screen sizes
    */
-  behavior?: 'responsive' | 'stacked';
+  behavior?: 'responsive' | 'stacked' | 'right';
```

Comment:
Can a button group be stacked and also aligned right?

Reply:
@csnizik @dierat yeah that's right, the `align` prop should be used to align something to the right.

Reply:
@dierat Same note as above here! https://github.com/chanzuckerberg/edu-design-system/pull/921/files#r838967593

Let's capture these things as tickets and address those things we catch in other work.